frameworks/MLNet/setup.sh

```diff
@@ -2,32 +2,32 @@
 HERE=$(dirname "$0")
 MLNET='mlnet'
 VERSION=${1:-"latest"}
+
 DOTNET_INSTALL_DIR="$HERE/lib"
 MLNET="$DOTNET_INSTALL_DIR/mlnet"
 DOTNET="$DOTNET_INSTALL_DIR/dotnet"
 SOURCE="https://mlnetcli.blob.core.windows.net/mlnetcli/index.json"
 
+export DOTNET_CLI_HOME="$DOTNET_INSTALL_DIR"
```

Comment:
if you export it, I guess it's used by `dotnet_install.sh`.
But what about the
```export DOTNET_ROOT="$DOTNET_INSTALL_DIR"```
is it needed?

Reply:
Yes, without it MLNet will not work: `A fatal error occurred. The required library libhostfxr.so could not be found.`